modules/k3s/build.gradle

```diff
@@ -3,7 +3,8 @@ description = "Testcontainers :: K3S"
 dependencies {
     api project(":testcontainers")
 
-    shaded 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.1'
+    // https://youtu.be/otCpCn0l4Wo
```

Comment:
lol 🤣

Reply:
Maybe we should say why, and under what circumstances this should be changed.

Have we disabled dependabot for this module/dependency combination yet?

Reply:
done! 👍

Reply:
We can keep Dependabot as the CI would fail now